auth_service/user_management/api/main.py

```diff
@@ -31,22 +34,34 @@
     TITLE,
     VERSION,
 )
+from auth_service.user_management.claims_repository.core.seed import seed_claims
 from auth_service.user_management.claims_repository.router import (
     router as claims_router,
 )
 from auth_service.user_management.user_registry.router import router as users_router
 
 configure_logging()
 
+
+@asynccontextmanager
+async def lifespan(app: FastAPI):  # pylint: disable=redefined-outer-name
+    """Setup the FastAPI application."""
```

Comment:
Why is this called lifespan? It is only for seeding claims, right?

Reply:
Because it's intended to be used aas lifespan function (contextmanager) for fastapi. We only make use of it as a way to initialize at startup, that's right.

Reply:
Oh, OK, I was not familiar with the FastAPI lifespan concept.